docs/index.md

```diff
@@ -3,14 +3,14 @@
 Invisible Internet Protocol (daemon)
 ====================================
 
-i2pd (I2P Daemon) is a full-featured C++ implementation of I2P client.
+i2pd (I2P Daemon) is a fully-featured C++ implementation of the I2P protocol.
```

Comment:
What about:

`i2pd (I2P Daemon) is a fully-featured I2P client implementation written in C++.`